packages/backend/src/plugins/inventory.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AggregatorInventory,
+  createRouter,
+  StaticInventory,
+} from '@backstage/plugin-inventory-backend';
+import type { PluginSandboxEnvironment } from '../types';
+
+export default async function ({ logger }: PluginSandboxEnvironment) {
+  const inventory = new AggregatorInventory();
+  inventory.enlist(
+    new StaticInventory([
+      { id: 'component1' },
+      { id: 'component2' },
+      { id: 'component3' },
+      { id: 'component4' },
+    ]),
+  );
+
+  return await createRouter({ inventory, logger });
+}
```

Comment:
The idea here is to do both the import, and the creation of related resources, and the router creation inside the sandbox. Currently the interface between the surroundings and this code is pretty basic (you pass in a logger, and you get out a router and nothing else), but we can let this grow in the future if the contact surface becomes larger.